.github/workflows/update-providers-ci-mgmt.yml

```diff
@@ -1,4 +1,50 @@
 name: Update Providers with new bridge version via ci-mgmt
 on:
+  release:
+    types: [published]
   workflow_dispatch:
+jobs:
+  get-bridge-latest-version:
+    runs-on: ubuntu-latest
+    steps:
+      - id: get-bridge-latest
+        uses: pozetroninc/github-action-get-latest-release@d1dafdb6e338bdab109e6afce581a01858680dfb
+        with:
+          repository: pulumi/pulumi-terraform-bridge
+          excludes: prerelease, draft
+      - id: bridge-version
+        run: |
+          version=$(echo ${{ steps.get-bridge-latest.outputs.release }} | cut -dv -f2)
+          echo "::set-output name=version::$version"
+    outputs:
+      bridge-latest: ${{ steps.bridge-version.outputs.version }}
+  get-sdk-latest-version:
+    runs-on: ubuntu-latest
+    steps:
+      - id: get-sdk-latest
+        uses: pozetroninc/github-action-get-latest-release@d1dafdb6e338bdab109e6afce581a01858680dfb
+        with:
+          repository: pulumi/pulumi
+          excludes: prerelease, draft
+      - id: sdk-version
+        run: |
+          version=$(echo ${{ steps.get-sdk-latest.outputs.release }} | cut -dv -f2)
+          echo "::set-output name=version::$version"
+    outputs:
+      sdk-latest: ${{ steps.sdk-version.outputs.version }}
+  dispatch:
+    runs-on: ubuntu-latest
+    needs: [get-bridge-latest-version, get-sdk-latest-version]
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.PULUMI_BOT_TOKEN }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'pulumi',
+              repo: 'ci-mgmt',
+              workflow_id: 'update-bridge-ecosystem-providers.yml',
+              ref: 'master',
+              inputs: {"bridge_version": "${{ needs.get-bridge-latest-version.outputs.bridge-latest }}", "sdk_version": "${{ needs.get-sdk-latest-version.outputs.sdk-latest }}", "automerge": true}
```

Comment:
@t0yv0 do you see anything wrong with this request? Failing with https://github.com/pulumi/pulumi-terraform-bridge/actions/runs/4326824183/jobs/7554718807 but the three parameters look like they match with https://github.com/pulumi/ci-mgmt/blob/master/.github/workflows/update-bridge-ecosystem-providers.yml#L11-L27

Reply:
It looks like `automerge` might need to be a string, despite the workflow dispatch config. https://github.com/cli/cli/issues/5246